packages/console-client/package.json

```diff
@@ -18,13 +18,13 @@
   },
   "author": "",
   "dependencies": {
+    "@microsoft/kiota-abstractions": "1.0.0-preview.51",
+    "@microsoft/kiota-serialization-json": "1.0.0-preview.51",
+    "@microsoft/kiota-serialization-text": "1.0.0-preview.48",
```

Comment:
Those lib should be at the same version?

Reply:
no the `text` one is a few numbers behind